src/web/app/pages-instructor/instructor-search-page/instructor-search-page.component.ts

```diff
@@ -66,6 +67,10 @@ export class InstructorSearchPageComponent implements OnInit {
 
       if (hasStudents) {
         this.studentsListRowTables = searchStudentsTable;
+        if (searchStudentsTable.length >= this.maxResultsToShow) {
+          this.statusMessageService.showWarningToast(`${this.maxResultsToShow} results have been shown on this page
+              but there may be more results not shown. Consider searching with more specific terms.`);
+        }
```

Comment:
Hi, I’m rather curious if this will ever be triggered when the result size is already limited by the backend const?
Would be good to have a snapshot too to show how this looks.